docs/build/messages/messages.md

```diff
@@ -345,70 +345,105 @@ for await (const page of conversation.messages(limit: 25)) {
 </TabItem>
 </Tabs>
 
-## Handle an unsupported content type error
+## Handle unsupported content types
 
-As more [custom](/docs/concepts/content-types#create-a-custom-content-type) and [standards-track](/docs/concepts/content-types#standards-track-content-types) content types enter the XMTP ecosystem, your app might receive a content type your app doesn't support. This could crash your app.
+As more [custom](/docs/concepts/content-types#create-a-custom-content-type) and [standards-track](/docs/concepts/content-types#standards-track-content-types) content types are introduced into the XMTP ecosystem, your app may encounter content types it does not support. This situation, if not handled properly, could lead to app crashes.
+
+Each content type includes a `contentFallback` property. This property provides a string that describes the expected value of the content type. Note that content fallbacks are immutable and are set by default in the protocol. If you are creating custom content types, you have the option to include a custom fallback. For more information on this, please visit the [Custom Content Type Tutorial](/docs/tutorials/custom-ct).
```

Comment:
>  and are set by default in the protocol

this seems a bit confusing. perhaps `and are set by default in network messages`. or maybe, `and are embedded into each message` or similar.

WDYT?